### PR TITLE
Add TextAlignment property to TextLayout

### PR DESCRIPTION
**Pull request by chkn**
This enables different text alignment when drawing multiline TextLayout (Width property set).

Includes Gtk, Mac, and WPF implementations.

### CODE REVIEW

**Comment by slluis** [comment]
It would be nice to have some test code.

File: Xwt/Xwt.Drawing/TextLayout.cs [hunk 1]
```diff
@@ -109,6 +110,10 @@ internal void InitForToolkit (Toolkit tk)
 					handler.SetWidth (Backend, width);
 				if (height != -1)
 					handler.SetHeight (Backend, height);
+				if (textAlignment != default(Alignment))
+					handler.SetAlignment (Backend, textAlignment);
+				if (textTrimming != default(TextTrimming))
```
Comment (sevoku) [inline Xwt/Xwt.Drawing/TextLayout.cs]: This doesn't seem to be related, or is it? Looks like it was just missing, then we should split it into an other commit (and bugfix PR).

**Comment by chkn** [comment]
@sevoku @slluis I've added a couple unit tests and an example in the samples app.

**Comment by sevoku** [comment]
Merged!
